experimental/s2ef_mapping.py

```diff
@@ -0,0 +1,45 @@
+from tqdm import tqdm
+import glob
+import ase.io
+import pickle
+import os
+import multiprocessing as mp
+import numpy as np
+
+
+def get_mapping(system):
+    randomid = os.path.basename(system)[:-7]
+    old_images = ase.io.read(system, ":")
+    old_energies = [image.get_potential_energy() for image in old_images]
```

Comment:
is looking at potential energy sufficient?
What if 2 frames have same potential energies?
Should we pair PE with some other aspect?

Reply:
Switched to positions instead, those will be unique and safer than energies.